pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -69,11 +69,11 @@ func (pc *PackageControllerClient) InstallController(ctx context.Context) error
 	}
 
 	if err = pc.ApplySecret(ctx); err != nil {
-		logger.Info("Warning: not able to create secret. Package installation might fail.", "error", err)
+		logger.Info("warning: no aws key/license provided, package controller will be installed but curated packages might not be installed")
```

Comment:
Lets run this by @binyamz for final wording. I dont have a strong opinion, just want it to be something that doesnt look so error-y. I also dont mind if this shows up under the default log output, could be a good way to call out the existence of curated packages, I just don't want people to mistake it for an actual error.

Reply:
Can keep it consistent with the current iam authenticator messaging. Recommend to change it to "Warning: No AWS key/license provided. Please be aware this will prevent the package controller from installing curated packages."